lambda/github_usage.py

```diff
@@ -161,12 +76,11 @@ def usage(message=None):
     """
     Compare registered users to org membership.
     """
-    access_token_param = os.environ.get("GITHUB_TOKEN")
-    access_token = get_ssm_param(access_token_param)
-    user_tokens = get_ssm_params(TOKEN_PREFIX)
+    user_tokens = config.get_ssm_params(TOKEN_PREFIX)
 
     org = os.environ.get("GITHUB_ORG")
-    members = get_github_org_members(org, access_token)
+    members = github_api.get_member_logins(github_api.get_github_org_members(org))
+
     member_count = len(members)
```

Comment:
`member_count` isn't needed, on line 88 you can just say `if members`

Reply:
I think this function could do with a rewrite to make it's intention clearer. However, I want to defer that for now.